test/e2e/functional_test.go

```diff
@@ -162,6 +162,17 @@ func (s *FunctionalSuite) TestEventOnPVCFail() {
 		})
 }
 
+func (s *FunctionalSuite) TestDefaultPDB() {
```

Comment:
do you need this test? you have a unit test below that looks the same

Reply:
deleted e2e test